translations/Marathi/special-terms.md

```diff
@@ -0,0 +1,42 @@
+﻿Terms
+
+Message : संदेश , Most appropriate synonym for general usage in zulip, <username> ला एक संदेश पाठवा
+Private Message (PM) : खाजगी संदेश , Best translation of private to imply it's meaning , <username> ला खासगी एक संदेश पाठवा
+Group PM : ग्रुप खाजगी संदेश  , Transliteration used for group , <group name> मध्ये एक ग्रुप खाजगी संदेश पाठवा
```

Comment:
Group has a different Word in Marathi, 'गट', which should be used instead of 'ग्रुप'

Reply:
Most of the users in the digital word would be familiar with the English word, therefore I chose transliteration

Reply:
@hrishi3108, @MadElf1337, I know Hindi so I am familiar with many of these terms and I think using the transliteration of `group` is better here. :)